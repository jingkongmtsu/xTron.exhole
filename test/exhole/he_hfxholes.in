##############
# Example for multipoint hfx hole with multithreads
#############

%global_infor
multi_threads true
cpu_threads_number 2
main spjobs
%end

%molecule
0 1
He   0.  0.   0.0
%end

%basis
g3large
%end

%xcfunc
name  hf
%end

%scf
max_scf_cycles  50
%end

%molecule
0 1
He   0.  0.   0.0
%end

%basis
g3large
%end

%parameters
xcints.xcoper true
xcints.xcprop hfxhole
xcints.nsvalues 2
xcints.1point 0.1     
xcints.1atom 0
xcints.holefile he_hfxholes.sv
XCInts.XCHoleAvg.dumpBatchHoles true
%end

%xcfunc
name  M_BR94CORR_OPP_X3
%end

%scf
max_scf_cycles  1
%end

%xcints
grid_points  3 1
threshold   1.0e-12
npt_cpu_thread 2
DO_XC_ENERGY_PROFILE true
%end

