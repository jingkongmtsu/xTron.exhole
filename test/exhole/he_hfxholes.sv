Coord:     0.000000      0.000000      5.272336
Rho  GAA  TA  LA
    0.00000005923267     0.00000000000004     0.00000015901972     0.00000060713751
    0.37500000000000     -0.00000007473237
    6.00000000000000     -0.00061963376273
Coord:     0.000000      0.000000      0.065091
Rho  GAA  TA  LA
    1.39121591611408    25.72901643566057     4.62347651030445  -179.27936980288987
    0.37500000000000     -0.42421026580810
    6.00000000000000     -0.00000000674660
Coord:     0.000000      0.000000      0.585815
Rho  GAA  TA  LA
    0.20154948535548     0.48780817303610     0.60507246170306     0.03584309073940
    0.37500000000000     -0.20035545546576
    6.00000000000000     -0.00000000998692
