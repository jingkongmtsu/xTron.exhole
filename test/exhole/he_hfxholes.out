
*******************************************
*     The Original Input File Content     *
*******************************************
##############
# Example for multipoint hfx hole with multithreads
#############

%global_infor
multi_threads true
cpu_threads_number 2
main spjobs
%end

%molecule
0 1
He   0.  0.   0.0
%end

%basis
g3large
%end

%xcfunc
name  hf
%end

%scf
max_scf_cycles  50
%end

%molecule
0 1
He   0.  0.   0.0
%end

%basis
g3large
%end

%parameters
xcints.xcoper true
xcints.xcprop hfxhole
xcints.nsvalues 2
xcints.1point 0.1     
xcints.1atom 0
xcints.holefile he_hfxholes.sv
XCInts.XCHoleAvg.dumpBatchHoles true
%end

%xcfunc
name  M_BR94CORR_OPP_X3
%end

%scf
max_scf_cycles  1
%end

%xcints
grid_points  3 1
threshold   1.0e-12
npt_cpu_thread 2
DO_XC_ENERGY_PROFILE true
%end


*******************************************
*            GlobalInfor Print            *
*******************************************
input   file is        : he_hfxholes.in
scratch file is        : /home/jing/scratch_xtron/7143-9771-460f-c615
Main func    is        : SPJOBS
number of CPU threads  : 2
we use multi-threads 
----------------------------------------------
          SINGLE POINT SCF CALCULATION        
----------------------------------------------
number of basis sets 10
DIIS error 0.2611109968    
HF iter is: 0, @@@@@ total energy is: -2.7505173167    
The most lowest energy so far is: -2.7505173167   , scf index is 0
NucRepulsion 0.0000000000    
core -3.9964661684   
JK   energy is: 1.2459488516    
XC   0.0000000000    
MTAG HF   0   0 -2.7505173167    0.2611109968    
DIIS error 0.0223012323    
HF iter is: 1, @@@@@ total energy is: -2.8585618263    
The most lowest energy so far is: -2.8585618263   , scf index is 1
NucRepulsion 0.0000000000    
core -3.9075539808   
JK   energy is: 1.0489921545    
XC   0.0000000000    
MTAG HF   1   1 -2.8585618263    0.0223012323    
DIIS error 0.0036689466    
HF iter is: 2, @@@@@ total energy is: -2.8599489368    
The most lowest energy so far is: -2.8599489368   , scf index is 2
NucRepulsion 0.0000000000    
core -3.8879795419   
JK   energy is: 1.0280306051    
XC   0.0000000000    
MTAG HF   2   2 -2.8599489368    0.0036689466    
DIIS error 0.0002672979    
HF iter is: 3, @@@@@ total energy is: -2.8599836349    
The most lowest energy so far is: -2.8599836349   , scf index is 3
NucRepulsion 0.0000000000    
core -3.8849373501   
JK   energy is: 1.0249537152    
XC   0.0000000000    
MTAG HF   3   3 -2.8599836349    0.0002672979    
DIIS error 0.0000234581    
HF iter is: 4, @@@@@ total energy is: -2.8599839179    
The most lowest energy so far is: -2.8599839179   , scf index is 4
NucRepulsion 0.0000000000    
core -3.8848111490   
JK   energy is: 1.0248272311    
XC   0.0000000000    
MTAG HF   4   4 -2.8599839179    0.0000234581    
DIIS error 0.0000001702    
HF iter is: 5, @@@@@ scf converged energy is: -2.8599839208    
The most lowest energy so far is: -2.8599839208   , scf index is 5
NucRepulsion 0.0000000000    
core -3.8847979564   
JK   energy is: 1.0248140356    
XC   0.0000000000    
MTAG HF   5   5 -2.8599839208    0.0000001702    
========================================
Dipole Moment(Debye) in post SCF process
========================================
dipole moment: 0.00000000       0.00000000       0.00000000     
total dipole moment: 0.00000000     
 0.764615s wall, 0.750000s user + 0.020000s system = 0.770000s CPU (100.7%)
-----------Second SCF begins-----------
iAtom 0

*******************************************
*        MolShellSize Object Print        *
*******************************************
Atomic number: 2     with radius:  17.203393   
Shell Radius:  1       2.905372    
Shell Radius:  2       5.577610    
Shell Radius:  3       10.357468   
Shell Radius:  4       17.203393   
Shell Radius:  5       4.471391    
Shell Radius:  6       8.783673    

WARN: nAngPts == 1
ubrxA -1.723  EXRA -1.672  ubrxB -1.723  EXRB -1.672  f  0.049
ubrxA -0.211  EXRA -0.190  ubrxB -0.211  EXRB -0.190  f  0.000
HOLE IS DONE
